### ISSUE TITLE
server can not boot 

### ISSUE THREAD

**Issue by jblingchen** [open-joinbase/JoinBase#15]
on my machine, the server can not boot with the command

the command is:
```bash
BASE_LOG=Debug ./server -c base.conf
```
stdout is:
```bash
Panic happened: Some(
    failed to start MQTT Runtime executor: InsufficientCpus { required: 6, available: 4 },
), 

Note: JoinBase server is designed to be fault-tolerant. It is enough to resolve this problem by just restarting the server processs.

    Welcome to report related messages to contact@joinbase.io for further improvement on the user experience.
```
the environment is:
```bash
Distributor ID:	Ubuntu
Description:	Ubuntu 22.10
Release:	22.10
Codename:	kinetic
```

**Comment by joinbase0** [comment]
@jblingchen thanks for reporting. The default number for MQTT exec pool in conf is larger than that available. This is OK to be fail fast. But it is a problem that the error message is not reported in the log.

**Comment by joinbase0** [comment]
we have fixed this problem in v2023.2.1 release. thanks!
